dwh/es_data_marts/README.md

```diff
@@ -1,15 +1,13 @@
-Welcome to your new dbt project!
+# dbt project for data marts
 
-### Using the starter project
+### Installing dependencies
 
-Try running the following commands:
-- dbt run
-- dbt test
+Run following commands:
+- ```python -m venv .venv```
+- ```source .venv/bin/activate```
+- ```pip install -r requirements.txt```
 
 
-### Resources:
-- Learn more about dbt [in the docs](https://docs.getdbt.com/docs/introduction)
-- Check out [Discourse](https://discourse.getdbt.com/) for commonly asked questions and answers
-- Join the [chat](https://community.getdbt.com/) on Slack for live discussions and support
-- Find [dbt events](https://events.getdbt.com) near you
-- Check out [the blog](https://blog.getdbt.com/) for the latest news on dbt's development and best practices
+### Run dbt:
```

Comment:
Whenever I try to run any `dbt` commands, such as `dbt test` or `dbt compile`, I get the same error:

```
Usage: dbt test [OPTIONS]
Try 'dbt test -h' for help.

Error: Invalid value for '--profiles-dir': Path '/home/ktsukanov/.dbt' does not exist.
```

I didn't dig deeper, but it seems there are some init/setup steps which should be added to this README

Reply:
addressed in [7b9065e](https://github.com/EMBL-EBI-ABC/PerturbationCatalogue/pull/263/commits/7b9065e6abb2197d9d48833b780567035b660d12)